### PR TITLE
ci(appveyor): update config

### PR DESCRIPTION
**Pull request by felixfbecker**
Testing whether this makes the build faster.

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -1,19 +1,21 @@
...
   - x64
 
 services:
   - mssql2016
 
-shallow_clone: true
+clone_depth: 1
 
 cache:
-  - node_modules
```
Comment (sushantdhiman) [inline appveyor.yml]: keep `node_modules` as well

Comment (eseliger) [reply appveyor.yml]: caching node_modules isn't safe, I would suggest caching only the npm cache

Comment (sushantdhiman) [reply appveyor.yml]: As long as we are caching modules some where I have no problem, main goal is fast build startup

**Comment by felixfbecker** [comment]
Decided to not fix what ain't broke. Not worth to investigate the timezone issue.

**Pull request closed without merging**